test/parallel/test-dns-onlookup.js

```diff
@@ -0,0 +1,68 @@
+// Flags: --expose-internals
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const { dnsException } = require('internal/errors');
+const { internalBinding } = require('internal/test/binding');
+const { getaddrinfo, GetAddrInfoReqWrap } = internalBinding('cares_wrap');
+const dns = require('dns');
+
+// Stub `getaddrinfo` to *always* error.
```

Comment:
??? Where is the stub?

Reply:
:hand: actually I forget to remove comment as this I saw(copy) in other testing file.